lib/xlat_tables_v2/xlat_tables_core.c

```diff
@@ -233,6 +233,98 @@ typedef enum {
 
 #if PLAT_XLAT_TABLES_DYNAMIC
 
+/*
```

Comment:
You'll have to take this outside of the #if.

Reply:
Thanks! fixed by taking those 2 helper functions outside of #if. I forgot to test it with PLAT_XLAT_TABLES_DYNAMIC=0.